core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -211,6 +213,11 @@ void stop() {
         return trackSelector.getAudioTracks(mediaPlayer);
     }
 
+    public List<PlayerSubtitleTrack> getSubtitleTracks() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
```

Comment:
I don't think we should have any logs in the library if I am completely honest. We should either forward to one of the `Player` listeners that will notify the client or throw. We should avoid this form of `Log` which is at odds with how the library operates.